plugin/src/test/java/io/jenkins/plugins/casc/misc/RoundTripAbstractTest.java

```diff
@@ -104,6 +104,9 @@ public void roundTripTest() throws IOException {
             String jenkinsConf = getResourceContent(resourcePath);
             assertConfigViaWebUI(jenkinsConf);
 
+            //Validate Schema before application
+            assertTrue(validateSchema(convertYamlFileToJson(this, "validJenkinsConfigurator.yml")));
```

Comment:
this should be the configuration being applied and not a static file in our repo.
also the api has now changed you should be asserting that there's no errors